src/Symfony/Component/Form/Extension/Core/EventListener/ResizeFormListener.php

```diff
@@ -102,6 +102,10 @@ public function preSubmit(FormEvent $event)
         $form = $event->getForm();
         $data = $event->getData();
 
+        if ($data instanceof \Traversable) {
+            @trigger_error('Support for Traversable is deprecated since version 3.1 and will be removed in 4.0. You should not use it anymore.', E_USER_DEPRECATED);
```

Comment:
I would say: `Use an array instead.` instead of `You should not use it anymore.`.